howto_docker.md

```diff
@@ -0,0 +1,58 @@
+# How to use Docker and Openfisca
+
+## Run OpenFisca on Docker
+
+1. Install Docker
```

Comment:
We should add a link to the appropriate Docker page, example https://docs.docker.com/install/